<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SingleEditor</class>
 <widget class="QWidget" name="SingleEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTextEdit" name="frontTextEdit"/>
     </item>
     <item>
      <widget class="QTextEdit" name="backTextEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="savePushButton">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
